testsuite/integration-arquillian/servers/auth-server/quarkus/src/main/content/conf/keycloak.properties

```diff
@@ -6,6 +6,10 @@ db.password = keycloak
 # Testsuite still relies on HTTP listener
 http.enabled=true
 
+# Disables strict hostname
+hostname.strict=false
+hostname.strict-https=false
```

Comment:
Was wondering if `hostname.strict-https` should be an option to proxy config rather than hostname config.

In production only https should be used right, so by default http is disabled. We can then see if http is enabled or not, and set strict-https based on that.

Now, with reverse proxy the proxy could talk to Keycloak over either https or http. If it uses https that's fine and we just have only https enabled, so will be forced to https. However, if proxy uses http to KC then in most cases it should set X-Forwarded-Proto, which would make KC know it's exposed on https, but if reverse proxy for some reason doesn't set X-Forwarded-Proto, we may want an optino on the proxy config to say `hey I know X-Forwarded-Proto` isn't set, but it should have been `https`.

Reply:
I don't know. But now you mentioned the correlation between this property and production, can we just remove it?

In production, we should always make sure URLs built by the server are using https scheme.

Reply:
Yeah, let's just hide the option, and make it always true in prod, without any way of changing it.

Reply:
@stianst Changed the property to `hidden" so that they are not available from help but still available for usage.